.github/workflows/test.yml

```diff
@@ -440,11 +441,10 @@ jobs:
         working-directory: crates/turbopack-ecmascript-runtime/js
         run: pnpm run check
 
-  # Very quick checks that we want to run first before all the long-running checks.
-  rust_pre_check:
+  rust_lint:
```

Comment:
This job just checks formatting and licences (and takes 5 minutes)

It shouldn't be a requirement for running checks